lib/roby/app.rb

```diff
@@ -235,7 +235,10 @@ def self.overridable_configuration(config_set, config_key, options = {})
         end
 
         # Allows to override the application base directory. See #app_dir
-        attr_writer :app_dir
+        def app_dir=(dir)
+            @app_dir = dir
+            @app_path = nil
```

Comment:
return the path set for consistency?
```suggestion
            @app_path = nil
            @app_dir = dir
```

Reply:
I will do it, because it is clearer, but know that `#=` functions in Ruby always return their argument. This is to never break an assignation chain, i.e. that in

```
a.some = b.other = 10
```

both a.some and b.other are always 10

![image](https://github.com/rock-core/tools-roby/assets/292/1c71bf8e-aacf-49ca-a8a0-dbae940077dc)